test/views/wallet/ConfigureMobilecoindView/MobilecoindDirectory.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import useMobilecoindConfigs from '../../../../app/hooks/useMobilecoindConfigs';
+import MobilecoindDirectory from '../../../../app/views/wallet/ConfigureMobilecoindView/MobilecoindDirectory';
+
+jest.mock('../../../../app/hooks/useMobilecoindConfigs');
+
+describe('MobilecoindDirectory', () => {
+  const mockUseMobilecoindConfigs = useMobilecoindConfigs as jest.MockedFunction<
+    typeof useMobilecoindConfigs
+  >;
+
+  // @ts-ignore mock
+  mockUseMobilecoindConfigs.mockImplementation(() => {
```

Comment:
This is fine for this test. But I would not get into the habit of setting mocks at the top scope this way unless you plan to actively use Jest's setup and teardowns https://jestjs.io/docs/en/setup-teardown. I prefer just writing a setup method. Either way is fine -- but what we have here could be dangerous if the text was expanded.

Reply:
Cool. I was going off https://testing-library.com/docs/react-testing-library/api/#cleanup
> cleanup#
> Unmounts React trees that were mounted with render.
> 
> Please note that this is done automatically if the testing framework you're using supports the afterEach global and it is injected to your testing environment (like mocha, Jest, and Jasmine). If not, you will need to do manual cleanups after each test.

I can still try to start implementing more setups either way.